src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -84,131 +87,141 @@ void set(Event event, String message) {
+
+		MessageType(String name, String pattern, Class<? extends Event>... events) {
 			this.name = name;
 			this.pattern = "[the] " + pattern;
 			this.events = events;
 		}
-		
+
 		static String[] patterns;
 		static {
 			patterns = new String[values().length];
 			for (int i = 0; i < patterns.length; i++)
 				patterns[i] = values()[i].pattern;
 		}
-		
+
 		@Nullable
-		abstract String get(Event e);
-		
-		abstract void set(Event e, String message);
-		
+		abstract String get(Event event);
+
+		abstract void set(Event event, String message);
+
 	}
-	
+
 	static {
 		Skript.registerExpression(ExprMessage.class, String.class, ExpressionType.SIMPLE, MessageType.patterns);
 	}
-	
-	@SuppressWarnings("null")
+
 	private MessageType type;
-	
-	@SuppressWarnings("null")
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
 		type = MessageType.values()[matchedPattern];
 		if (!getParser().isCurrentEvent(type.events)) {
-			Skript.error("The " + type.name + " message can only be used in a " + type.name + " event", ErrorQuality.SEMANTIC_ERROR);
+			Skript.error("The " + type.name + " message can only be used in a " + type.name + " event");
 			return false;
 		}
 		return true;
 	}
-	
+
 	@Override
-	protected String[] get(final Event e) {
-		for (final Class<? extends Event> c : type.events) {
-			if (c.isInstance(e))
-				return new String[] {type.get(e)};
+	protected String[] get(Event event) {
+		for (Class<? extends Event> c : type.events) {
+			if (c.isInstance(event))
+				return new String[] {type.get(event)};
 		}
 		return new String[0];
 	}
-	
+
 	@Override
 	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
+	public Class<?>[] acceptChange(ChangeMode mode) {
 		if (mode == ChangeMode.SET)
 			return CollectionUtils.array(String.class);
 		return null;
 	}
-	
+
 	@Override
-	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) {
+	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
 		assert mode == ChangeMode.SET;
 		assert delta != null;
-		for (final Class<? extends Event> c : type.events) {
-			if (c.isInstance(e))
-				type.set(e, "" + delta[0]);
+		for (Class<? extends Event> c : type.events) {
+			if (c.isInstance(event))
+				type.set(event, "" + delta[0]);
 		}
 	}
-	
+
 	@Override
 	public boolean isSingle() {
 		return true;
 	}
-	
+
 	@Override
 	public Class<? extends String> getReturnType() {
 		return String.class;
 	}
-	
+
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
-		return "the " + type.name + " message";
+	public String toString(@Nullable Event event, final boolean debug) {
+		return type.name + " message";
```

Comment:
Why did you make this change?

Reply:
TPgames had a rule to not include `the` in to strings because another pattern may add it, and the concatenation of another syntax could make it weird.